bin/installer.sh

```diff
@@ -339,9 +341,16 @@ function install_chef_client {
         esac
     done
 
+
     if [[ $chef_install == "yes" ]]; then
+        echo "Enter the chef-client version you would like to install or press ENTER for $DEFAULT_CHEF_VERSION"
+        chef_version=$( read_terminal )
         echo "Installing chef-client."
-        curl -L http://www.opscode.com/chef/install.sh | sudo bash -s -- -v 12.6
+        if [ "X$chef_version" == "X" ]; then
```

Comment:
how about here do:
chef_version=$DEFAULT_CHEF_VERSION

then always call out to curl | bash with the same line?  That way if other options change they only need to change in 1 place.